packages/formation/sass/modules/_m-button.scss

```diff
@@ -125,37 +122,52 @@ button.short {
 
 // Opens / closes a sidebar navigation on detail pages.
 .va-btn-sidebarnav-trigger {
-  background: $color-gray-lightest;
-  border-radius: 0;
-  display: block;
-  color: $color-link-default;
-  margin: 1rem 0 2rem;
-  text-align: left;
+  margin: 0;
+  padding: 0;
 
-  span {
-    align-items: center;
-    display: flex;
+  &.fixed-trigger {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: $top-layer - 1;
```

Comment:
Once the position of the trigger gets `fixed` its z-index comes into play and we need to make sure that it's _behind_ the sidenav menu itself.